### ISSUE TITLE
My articles not show correctly,show the source code ,why?

### ISSUE THREAD

**Issue by luoyedaren** [itorr/itorr.github.io#8]
{{#.}}
	<article>
		<span class="num"></span>
		<span class="size">{{size}}</span>
		<h3><a href="#/{{url}}">{{title}}</a></h3>
	</article>
	{{/.}}

**Comment by solarhell** [comment]
hey guy, [this](https://github.com/luoyedaren/chenghai.blog) is your repo, right?
Please create a new repo named like 'luoyedaren.github.io' and push your file in master branch, then it will works.

**Comment by luoyedaren** [comment]
so glad  to see your message ,this is my site created by your way :https://luoyedaren.github.io/chenghai.blog/. is it correct?   and it's not show my articles .Could you help to see it?

**Comment by luoyedaren** [comment]
the <xmp> tags not works very well, I supposed.

**Comment by solarhell** [comment]
But i can't see you have a repo named 'luoyedaren.github.io'.
This project is just an example that the username is 'itorr' and he has an repo named 'itorr.github.io'.

**Comment by luoyedaren** [comment]
it must be named 'luoyedaren.github.io' for repo, Mine repo's name is "chenghai.blog"  ,Maybe the reason is this.Thank you .let me try it!

**Comment by luoyedaren** [comment]
https://luoyedaren.github.io/  ,fix the name ,but not work...

**Comment by solarhell** [comment]
Please replace the static assets to your source cause you are using https.
You can check the error in Chrome's 'inspect'.

**Comment by luoyedaren** [comment]
![image](https://cloud.githubusercontent.com/assets/7297796/24131005/07bc6c5c-0e27-11e7-87a5-ca4c85966cc5.png)
my aasets was in already

**Comment by solarhell** [comment]
Please check line 37 in index.html and line 109 in 'i/gitblog.js'.

**Comment by luoyedaren** [comment]
fastclick.m.js mustache.min.js   not in your repo..

**Comment by solarhell** [comment]
I think download the js file and push to the repo is not so difficult to explain.

**Comment by luoyedaren** [comment]
yeah   ,guys    you are right ..  I'm not goods at html ,js . Hope not too much js needed...  Thank you very much

**Comment by solarhell** [comment]
You also need to change the index.html to
``` javascript
var
gitConfig={
	ssh:'git@github.com:luoyedaren/luoyedaren.github.io.git'
}
```

**Comment by luoyedaren** [comment]
fixed .thank for you advice  . but   not the end ...hh   .
itorr2.js:289 Mixed Content: The page at 'https://luoyedaren.github.io/' was loaded over HTTPS, but requested an insecure script 'http://1.mouto.org/x.js'. This request has been blocked; the content must be served over HTTPS.
blocked!  how to fix it

**Comment by solarhell** [comment]
我刚提了2个PR。
另外， 'x.js' 是坨坨肉的统计代码，直接去掉或者换成自己的就行了。
HTTPS下，所有资源默认都应该通过HTTPS加载。

**Comment by luoyedaren** [comment]
well done! Thank you . I'am java coder  . Nice to know you.

**Comment by solarhell** [comment]
我是写php的

**Comment by luoyedaren** [comment]
![image](https://cloud.githubusercontent.com/assets/7297796/24132582/8e42d0e0-0e31-11e7-9575-d8c1f3580431.png)
fall in a hole....

**Comment by solarhell** [comment]
http://front.dog/smartisan/i/md.js  这个需要加载一下这个静态文件 你可以放到自己的repo里用https来加载
可以加入这个QQ群 311084414
